cassandraprotocol/constants.go

```diff
@@ -3,12 +3,16 @@ package cassandraprotocol
 type ProtocolVersion = uint8
 
 const (
-	ProtocolVersion3    = ProtocolVersion(0x3)
-	ProtocolVersion4    = ProtocolVersion(0x4)
-	ProtocolVersion5    = ProtocolVersion(0x5)
-	ProtocolVersionMin  = ProtocolVersion3
-	ProtocolVersionMax  = ProtocolVersion4
-	ProtocolVersionBeta = ProtocolVersion5
```

Comment:
Had to remove Min, Max and Beta, it doesn't work with DSE versions. I replaced these with functions in util.go.